content/en/docs/concepts/storage/volumes.md

```diff
@@ -462,7 +462,7 @@ spec:
     required:
       nodeSelectorTerms:
       - matchExpressions:
-        - key: failure-domain.beta.kubernetes.io/zone
+        - key: topology.kubernetes.io/zone
```

Comment:
Add a comment that this was added in 1.21/1.22. Releases before then still have to use the beta label.

Reply:
Done